changelog.md

```diff
@@ -1,3 +1,15 @@
+# [7.3.0](https://github.com/TehShrike/abstract-state-router/releases/tag/v7.3.0)
+
+Removes the concept of resetting states.
+
+The concept of resetting breaks down if your component library doesn't support
+- slots
+- resetting the state of a component without resetting the contents of slots
+
+For renderers that "reset" states by destroying the existing component and re-constructing it, stuff would break in any case where a parent and child state were both told to reset at once.  Whenever the parent would reset and destroy its part of the DOM, the child would get wiped out.
+
+Existing renderers don't need to change to work with this version of ASR, it's just that their `reset` function won't get called any more.
```

Comment:
Might be good to mention in here that the `beforeResetState` and `afterResetState` events are gone

Reply:
which kinda seems like a breaking change, if those events stop firing 😬